packages/config/src/class-textdomain-customizer.php

```diff
@@ -203,7 +203,7 @@ private function customize_textdomain_in_file( $file_path ) {
 		$file_contents = file_get_contents( $file_path );
 
 		$file_contents = str_replace(
-			'\'JETPACK_CUSTOMIZE_TEXTDOMAIN\'',
+			'JETPACK_CUSTOMIZE_TEXTDOMAIN',
 			'\'' . $this->textdomain . '\'',
```

Comment:
Let's remove quotes here too, so we don't end up duplicating them.
```suggestion
			$this->textdomain,
```

Reply:
Oh, wait, nevermind, I see now that you have removed the quotes in the code as well.